kit/inputs/src/UIMaterialTextView/useClearButton.tsx

```diff
@@ -11,10 +11,10 @@ export function useClearButton(
     inputHasValue: boolean,
     isFocused: boolean,
     isHovered: boolean,
-    clear: () => void,
+    clear?: () => void,
```

Comment:
Without current context it can be so not obvious how clear button can be created without it IMO

Reply:
@sertony FYI

Reply:
That's the thing, that the clear button SHOULD NOT be created without it ;) I like the idea